docs/usage/configuration-options.md

```diff
@@ -3617,10 +3617,10 @@ Follow these steps:
 1. If you're running Renovate in app mode: make sure the app has `read` permissions for "Dependabot alerts".
    If you're the account administrator, browse to the app (for example [the Mend Renovate App](https://github.com/apps/renovate)), select "Configure", and then scroll down to the "Permissions" section and make sure that `read` access to "Dependabot alerts" is mentioned
 
-Once the above conditions are met, and you got one or more vulnerability alerts from GitHub for this repository, then Renovate tries to raise fix PRs.
+Once the above conditions are met, _and_ if GitHub finds a Dependabot alert for this repository, then Renovate tries to create a fix PR.
```

Comment:
This doesn't sound quite right because GitHub _is_ Dependabot and vice versa. I would prefer a passive tense to this

Reply:
I've reverted to what we had before. Open for suggestions though! 😉